src/ports/postgres/modules/glm/glm.py_in

```diff
@@ -350,7 +351,7 @@ def __glm_compute(schema_madlib, tbl_source, tbl_output, col_dep_var, col_ind_va
                    **args))
 
     # clean up
-    plpy.execute("""DROP TABLE IF EXISTS pg_temp.{rel_state} """.format(**args))
```

Comment:
Do we call this only once? Since this isn't defined as a temp table, we probably would want to reclaim the space of table, similar to the DL module using `dev_opt_unsafe_truncate_in_subtransaction` guc. (https://github.com/apache/madlib/blob/master/src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.py_in#L985-L988)

Reply:
Yes this is called only once at the end of the training.